pom.xml

```diff
@@ -202,6 +202,8 @@
     <testJavaHome>${env.JAVA_HOME}</testJavaHome>
     <skipOsgiTestsuite>false</skipOsgiTestsuite>
     <skipAutobahnTestsuite>false</skipAutobahnTestsuite>
+    <!--avoid below packet indexed twice-->
```

Comment:
`avoid below packet indexed twice` -> should this say `package`? Also if it is excluded, how can we be sure it runs at least once?